modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc

```diff
@@ -48,134 +51,160 @@ sudo systemctl restart httpd
 
-NOTE: If you are running ownCloud on a 32-bit system, any `open_basedir` directive in your `php.ini` file needs to be commented out.
+=== Apache Directives
 
-Set the following two parameters inside `php.ini`, using your own desired file size values, as in the following example:
+* {limitrequestbody-url}[LimitRequestBody]
+* {sslrenegbuffersize-url}[SSLRenegBufferSize]
 
-[source]
-----
-upload_max_filesize = 16G
-post_max_size = 16G
-----
+=== Apache with mod_reqtimeout
 
-Tell PHP which temp file you want it to use:
+The {mod_reqtimeout-url}[mod_reqtimeout] Apache module could also stop large uploads from completing. If you're using this module and getting failed uploads of large files, either disable it in your Apache config or raise the configured `RequestReadTimeout` timeouts.
 
-[source]
+Disable mod_reqtimeout On Ubuntu::
++
+On Ubuntu, you can disable the module by running the following command:
++
+[source,console]
 ----
-upload_tmp_dir = /var/big_temp_file/
+sudo a2dismod reqtimeout
 ----
 
-*Output Buffering* must be turned off in `.htaccess` or `.user.ini` or `php.ini`, or PHP will return memory-related errors:
-
-[source]
+Disable mod_reqtimeout On CentOS::
++
+On CentOS, comment out the following line in `/etc/httpd/conf/httpd.conf`:
++
+[source,apache]
 ----
-output_buffering = 0
+LoadModule reqtimeout_module modules/mod_reqtimeout.so
 ----
++
+When you have done run `asdismod` or updated `/etc/httpd/conf/httpd.conf`, restart Apache.
++
+TIP: There are also several other configuration options in your web server config which could prevent the upload of larger files. Please see your web server's manual, for how to configure those values correctly:
 
-== Configuring ownCloud
-
-As an alternative to the `upload_tmp_dir` of PHP (e.g., if you don't have access to your `php.ini`) you can also configure a temporary location for uploaded files by using the `tempdirectory` setting in your `config.php`.
-
-If you have configured the `session_lifetime` setting in your `config.php`. 
-See xref:configuration/server/config_sample_php_parameters.adoc[Sample Config PHP Parameters], to make sure it is not too low. 
-This setting needs to be configured to at least the time (in seconds) that the longest upload will take.
-If unsure, remove this entirely from your configuration to reset it to the default shown in the `config.sample.php`.
+=== Apache with mod_fcgid
 
-== General Upload Issues
+* {fcgidmaxrequestinmem-url}[FcgidMaxRequestInMem]
+* {fcgidmaxrequestlen-url}[FcgidMaxRequestLen]
 
-Various environmental factors could cause a restriction of the upload size. 
-Examples are:
+WARNING: If you are using Apache/2.4 with mod_fcgid, as of February/March 2016, `FcgidMaxRequestInMem` still needs to be significantly increased from its default value to avoid the occurrence of segmentation faults when uploading big files. This is not a regular setting but serves as a workaround for {mod_fcgid_bug_51747-url}[Apache with mod_fcgid bug #51747].
```

Comment:
```suggestion
WARNING: If you are using Apache 2.4 with mod_fcgid, as of February/March 2016, `FcgidMaxRequestInMem` still needs to be significantly increased from its default value to avoid the occurrence of segmentation faults when uploading big files. This is not a regular setting but serves as a workaround for {mod_fcgid_bug_51747-url}[Apache with mod_fcgid bug #51747].
```